build/nuget/Uno.WinUI.RemoteControl.nuspec

```diff
@@ -26,5 +26,17 @@
 	<files>
 		<file src="..\..\src\Common\uno.png" target="/" />
 
+		<file src="_._" target="lib\net8.0" />
```

Comment:
What does "_,_" mean? it's the first time I see this

Reply:
It's a nuget trick that forces a target framework support for a nuget package, while adding no references. Without it, a nuget package is not compatible with any target framework and fails to be referenced.